docs/content/en/blog/articles/why-k8ssandra/index.md

```diff
@@ -52,7 +52,7 @@ Another example is how K8ssandra leverages TCP [ingresses](https://kubernetes.io
 
 ## Optimizing Developer Productivity, Cost, and Performance
 
-While fast access to CQL endpoints is attractive to developers who are already using Cassandra, most developers we’ve talked with would prefer to code to data APIs for new development, rather than learning a new database query language. This is why K8ssandra also includes [K8ssandra also includes Stargate](/docs/topics/stargate/), a data services gateway, which provides REST, Document, and GraphQL APIs on top of Cassandra. These familiar APIs lead to increased productivity and reduced time to market.
+While fast access to CQL endpoints is attractive to developers who are already using Cassandra, most developers we’ve talked with would prefer to code to data APIs for new development, rather than learning a new database query language. This is why[K8ssandra also includes Stargate](/docs/topics/stargate/), a data services gateway, which provides REST, Document, and GraphQL APIs on top of Cassandra. These familiar APIs lead to increased productivity and reduced time to market.
```

Comment:
Missing a space between the link and `why`

```suggestion
While fast access to CQL endpoints is attractive to developers who are already using Cassandra, most developers we’ve talked with would prefer to code to data APIs for new development, rather than learning a new database query language. This is why [K8ssandra also includes Stargate](/docs/topics/stargate/), a data services gateway, which provides REST, Document, and GraphQL APIs on top of Cassandra. These familiar APIs lead to increased productivity and reduced time to market.
```